Upstream: Unknown
Source: Gentoo
Summary: Don't install .a files in Python site-packages directories.

--- libxslt-1.1.27.orig/python/Makefile.am
+++ libxslt-1.1.27/python/Makefile.am
@@ -24,7 +24,8 @@
 	libxslt-python-api.xml	\
 	$(DOCS)
 
-libxsltmod_la_LDFLAGS =  $(WIN32_EXTRA_LDFLAGS) -module -avoid-version
+libxsltmod_la_CPPFLAGS = -shared
+libxsltmod_la_LDFLAGS = $(WIN32_EXTRA_LDFLAGS) -module -avoid-version -shared
 
 if WITH_PYTHON
 mylibs = \
